transformer_engine/pytorch/attention.py

```diff
@@ -735,6 +760,7 @@ def forward(
         # Pre-allocate memory for key-values for inference.
         # =================================================
 
+        is_first_step = False
```

Comment:
If I understand this correctly, would a better name be `is_first_inference_step`?